packages/react-components/src/components/styles/VideoGallery.styles.ts

```diff
@@ -27,7 +27,7 @@ export const floatingLocalVideoModalStyle: IStyleFunctionOrObject<IModalStylePro
     boxShadow: theme.effects.elevation8,
     borderRadius: theme.effects.roundedCorner4,
     position: 'absolute',
-    bottom: '1rem',
+    bottom: '5rem',
```

Comment:
I see this also in the meeting composite where it overlaps the side pane. Is it possible to position this relative to the parent instead?

normally you can set `position: relative` on the parent and the absolute position will use the parent instead of the page here

Reply:
Having the padding bottom also means if the control bar changes this will be incorrect again and if the videogallery is used outside of a control bar it will randomly be floating:
![image](https://user-images.githubusercontent.com/2684369/126804913-ee22e19c-eaf3-4814-9160-b57c5c8401b3.png)

(though that could be fixed by passing in value from the composite instead of hardcoding this at the component layer - but if possible it would be better if we could set this relative to the parent so it should move if the parent changes size)

Reply:
Updated, needed to apply a layout as the models by default are rendered at the end of the document, now its rendered inside the video gallery component

Results (the video tile is positioned relative to the video gallery - even when the text box is dynamically resized at runtime):
| test | result |
| -- | -- |
| content to the right hand side | ![image](https://user-images.githubusercontent.com/2684369/126843045-912c6a10-e6da-4117-a430-ee920c16972e.png) |
| content below | ![image](https://user-images.githubusercontent.com/2684369/126843128-c21a6497-cd65-4c17-afc4-e89ded73158d.png) |

Reply:
When testing meeting composite, correctly did not overlap control bar or side pane when opened:
![image](https://user-images.githubusercontent.com/2684369/126844154-191b3466-e2c5-460d-b9e2-8b9aee8284bc.png)